common/src/config.rs

```diff
@@ -199,6 +199,9 @@ impl Default for Metrics {
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(default)]
 struct Limits {
+    /// How many events can be sent concurrently before Relay starts dropping. This is a limit
+    /// independent of and in addition to any rate limits upstream enforces.
+    max_active_events: usize,
```

Comment:
Can we do `max_concurrent_events` here?

Reply:
This could even include something like “buffer”, as this is essentially the behavior we emulate.

Reply:
I'm not sure if `buffer` is descriptive of what it does, opting for `max_concurrent_events`